tests/sparseml/onnx/helpers.py

```diff
@@ -992,50 +994,35 @@ def analyzer_models(request):
     scope="session",
     params=[
         (
-            {
-                "domain": "cv",
-                "sub_domain": "classification",
-                "architecture": "resnet_v1",
-                "sub_architecture": "50",
-                "framework": "pytorch",
-                "repo": "sparseml",
-                "dataset": "imagenet",
-                "training_scheme": None,
-                "sparse_name": "base",
-                "sparse_category": "none",
-                "sparse_target": None,
-            },
+            "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/base-none",
             "resnet50",
         ),
         (
-            {
-                "domain": "cv",
-                "sub_domain": "classification",
-                "architecture": "mobilenet_v1",
-                "sub_architecture": "1.0",
-                "framework": "pytorch",
-                "repo": "sparseml",
-                "dataset": "imagenet",
-                "training_scheme": None,
-                "sparse_name": "base",
-                "sparse_category": "none",
-                "sparse_target": None,
-            },
+            "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/base-none",  # noqa 501
             "mobilenet",
         ),
     ],
 )
 def onnx_repo_models(request) -> OnnxRepoModelFixture:
-    model_args, model_name = request.param
-    model = Zoo.load_model(**model_args)
-    model_path = model.onnx_file.downloaded_path()
-    data_paths = [data_file.downloaded_path() for data_file in model.data.values()]
+    # clear cache
+    CACHE_DIR = os.path.expanduser(os.path.join("~", ".cache", "sparsezoo"))
```

Comment:
should avoid clearing cache especially for tests as this can get very time consuming. in general shouldn't delete the entire user cache without the user asking for this

Reply:
This ugly hack will be gone once https://github.com/neuralmagic/sparsezoo/pull/183 lands.